baseapp/datadog/datadog.go

```diff
@@ -68,9 +68,10 @@ func StartEmitter(s *baseapp.Server, c Config) error {
 	if err != nil {
 		return errors.Wrap(err, "datadog: failed to create client")
 	}
-	client.Tags = append(client.Tags, c.Tags...)
```

Comment:
DD statsd client no longer has a built-in tags feature, so had to move this to the emit logic.

Reply:
This is actually still supported, but is now part of `statsd.New`:

```go
client, err := statsd.New(c.Address, statsd.WithTags(c.Tags))
```

Reply:
ah, will correct.

Reply:
Corrected; feels like we should hold on identifying the oddities James ran into tho :-)